### PR TITLE
Update message keys for dedicated client scope role mapping text.

### LINKED ISSUE #27101: Translation keys for dedicated scope role mappings are incorrect
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

Translation keys for dedicated scope role mappings are incorrect.

This results in the key names being shown on the UI instead of any more useful text.

image

From Keycloak 22:

image

### Version

23, nightly

### Regression

- [X] The issue is a regressions

### Expected behavior

Appropriate help text is displayed

### Actual behavior

Translation key names are displayed.

### How to Reproduce?

1) Start Keycloak
2) Go to admin console, choose any client (`account` works)
3) Click scopes
4) Select the "dedicated" scope
5) Click the "Scope" tab
6) Ensure "Allow full scope is off"
7) Be presented with the translation key message

### Anything else?

Functionally still works, just not as great UX (user must look at Keycloak docs instead of the UI for context).

Seems to surface from here:

https://github.com/keycloak/keycloak/blob/f4c8ded5f6a8db1f1bd23285b4f04b894aa5e468/js/apps/admin-ui/src/clients/scopes/DecicatedScope.tsx#L113

Seems like it was meant to be translated to `noRoles-client` and `noRolesInstructions-client` based on this documentation
https://github.com/keycloak/keycloak/blob/7bde7c30cc1265df8eed7034760c418eb78b462c/docs/documentation/upgrading/topics/keycloak/changes-23_0_0.adoc#L275-L276

### PR DESCRIPTION
**Pull request by jsorah**
Closes keycloak/keycloak#27101

### CODE REVIEW

**Review by edewit — DISMISSED** [review]

**Comment by jsorah** [comment]
Ah, I see why this is failing - seems the message was re-used in two places. I'll put the old key back and keep the new ones I created by renaming it.

**Comment by jsorah** [comment]
DCO check looks like its failing, possibly due to my rebase when I put the old keys back - main has one commit newly added without the DCO.  LMK if I should try recreating the PR.

**Comment by edewit** [comment]
@jsorah can you rebase with the `-s`?

**Comment by jsorah** [comment]
Looks like using GitHub's rebase option fixed the DCO issue - doing it from the CLI on my machine did not resolve it.

Should be good to review now @edewit

File: js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties
Comment (jonkoops) [inline js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties:1]: @miquelsi @rmartinc @Aboullos can I ask one of you to put in a review for the Spanish translations?

File: js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_pl.properties
Comment (jonkoops) [inline js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_pl.properties:1]: @agagancarczyk could you review these Polish translations?

File: js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_zh_CN.properties
Comment (jonkoops) [inline js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_zh_CN.properties:1]: @xianli123 could I ask you to verify these Chinese translations?

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
@jsorah if this is updating some message keys, should the old ones not be removed?

**Comment by jsorah** [comment]
> @jsorah if this is updating some message keys, should the old ones not be removed?

It's actually an interesting situation - I discovered that the keys were actually duplicated between the two screens before, hence why I kept the old ones as to not break the other screen. Contextually, they are different though, which is why I modified the English translation key slightly. https://github.com/keycloak/keycloak/issues/27101#issuecomment-1952549201

**Review by miquelsi — APPROVED** [review]
Approving Spanish translation

File: js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_pl.properties
Comment (agagancarczyk) [reply js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_pl.properties:1]: All looks good!

**Review by agagancarczyk — APPROVED** [review]

File: js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_zh_CN.properties
Comment (xianli123) [reply js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_zh_CN.properties:1]: Looks good to me. Thanks! @jonkoops

**Review by xianli123 — APPROVED** [review]

**Pull request merged**